sessions/biases-in-delay-distributions.qmd

```diff
@@ -217,12 +227,16 @@ Can you establish under which conditions the bias in estimation gets worse?
 # Real-time estimation: truncation
 
 The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
-However, information on delay distribution is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+
+Estimating delays in real time comes with particular challenges, as we know that our data are usually incomplete: not enough time has passed to observe all of the relevant events that could have happened.
+We call this incompleteness *truncation*.
+This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events (such as infections) - our estimates might be biased because we are missing some of the data to get the full picture.
```

Comment:
I think this truncation def could do with a workshop. The key is needing to mention both events I.e the event we are using and the observed event. It's when we condition on a delay observed event we get right truncation of the target event.

Reply:
see what you think in [commit](https://github.com/nfidd/nfidd/pull/450/commits/637f81898ed59c9b4065c320ef02ac451389a0d1), or please do edit away!